syntax = "proto3";
package power_data;
import "google/protobuf/timestamp.proto";

message PowerLog{
    google.protobuf.Timestamp timestamp = 1;            //データを取得した時間のタイムスタンプ
    int32 Switching_Power_Input_mA = 2; //スイッチング電源
    int32 Switching_Power_Input_mV = 3;
    int32 Battery_Input_mA = 4;         //バッテリー
    int32 Battery_Input_mV = 5;
    int32 SBC_Power_Supply_mA = 6;      //Jetsonの電源
    int32 SBC_Power_Supply_mV = 7;
    int32 Actuator_Power_Supply_mA = 8; //サーボ
    int32 Actuator_Power_Supply_mV = 9;
}